app/controllers/v0/upload_supporting_evidences_controller.rb

```diff
@@ -4,23 +4,69 @@
 
 module V0
   class UploadSupportingEvidencesController < ApplicationController
-    include FormAttachmentCreate
     extend Logging::ThirdPartyTransaction::MethodWrapper
     service_tag 'disability-application'
 
-    FORM_ATTACHMENT_MODEL = SupportingEvidenceAttachment
+    VALID_FILE_CLASSES = [ActionDispatch::Http::UploadedFile, Rack::Test::UploadedFile].freeze
 
     wrap_with_logging(
       :save_attachment_to_cloud!,
       additional_class_logs: {
         form: '526ez supporting evidence attachment',
-        action: "upload: #{FORM_ATTACHMENT_MODEL}",
+        action: 'Uploaded user-provided attachment to S3',
         upstream: 'User provided file',
         downstream: "S3 bucket: #{Settings.evss.s3.bucket}"
       },
       additional_instance_logs: {
         user_uuid: %i[current_user account_uuid]
       }
     )
+
+    def create
```

Comment:
@NB28VT would you mind explaining why this `create` isn't feature-gated? Also, is the intention to have it create both EVSS and LH supporting evidence attachment before the flag is flipped?

Reply:
Piggybacking on this comment

> Creates a new LighthouseSupportingEvidenceAttachment class that inherits from FormAttachment. This was required to solve an inheritance issue that is well documented in the [commit message](https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/7e5c4658bb3877539f4f9fcad82cae95f5421bcd) for this change. Note I also had to create Lighthouse verisons of the superclasses and associated services for this attachment subtype, which mirrors the behavior in SupportingEvidenceAttachment

Can you elaborate on why a new model was needed? I see you link to the commit message, but I can't view it